source/docs/advanced/payments/payment-on-account.md

```diff
@@ -0,0 +1,38 @@
+---
+id: payment-on-account
+title: Payment on account
+---
+
+Payment on account is a payment method for B2B context that allows companies to make purchases up to the credit limit that is specified in their profile.
+
+<blockquote class="feature--new">
+_Since version 2.11.0_
+</blockquote>
+
+## Enable Payment on account
```

Comment:
For consistency with other payment methods, I think we should add a new title level with the Platform.

```
## Magento2 B2B

<blockquote class="feature--new">
_Since version 2.11.0_
</blockquote>

### Enable Payment on account
```